.github/workflows/CD.yml

```diff
@@ -0,0 +1,16 @@
+name: 🪂 Continuous Deployment
+
+on:
+  push:
```

Comment:
I'm not sure all these triggers are necessary. We've had an undocumented convention to use git tags for new version releases of apps, so I guess we can trigger this only upon pushing new tags.

Reply:
From a _"Continuous"_ deployment perspective, I've found it is handy to have the images built and pushed more often.

## `on: push: master`
So we can easily deploy bugfixes without versioning a new tag number.

(Side note, we should change `master` to `main` here and in all `apps-*`)

## `pull_request: master`
This is helpful for testing the image of a new feature, before pulling it in.

## `releases`
If clamsproject is not using releases, we can remove this, but we should probably issue regular releases for major stable versions.

## `tags`
We can add triggers on tags, e.g:
```
    tags:
      - '*'
```
... to build on any tag push.

I'd still like to keep the other triggers in for now. If we need to prune old images in the future, we can create a workflow Action for that as well.